src/picongpu/include/particles/Particles.kernel

```diff
@@ -86,7 +87,13 @@ __global__ void kernelCloneParticles(ParticlesBox<MYFRAME, simDim> myBox, Partic
         PMACC_AUTO(parDest, ((*myFrame)[threadIdx.x]));
         PMACC_AUTO(parSrc, ((*frame)[threadIdx.x]));
         assign(parDest, parSrc);
+
+        manipulateFunctor(block + DataSpaceOperations<simDim>::map<SuperCellSize>(threadIdx.x),
```

Comment:
We need to remove the guarding cells.
`... - mapper.getGuardingSuperCells() * SuperCellSize::toRT()` is missing.